### DISCUSSION TITLE
Got returns a different response than other http request libraries/curl

### DISCUSSION THREAD

**Discussion by sbalaji1996** [sindresorhus/got#1661] [Q&A] [answered]
#### Describe the bug

 - Node.js version: v14.16.0
 - OS & version: macOS Big Sur v11.2.2, `got` version 11.8.2

I'm attempting to make GET requests to the Pipedrive API ([docs](https://pipedrive.readme.io/docs/core-api-concepts-about-pipedrive-api)), located at `'https://api.pipedrive.com/'`. However, regardless of the path and query parameters I add to the request url, `got` returns a 403 response. Oddly, other node http request libraries, specifically [`http`](https://nodejs.org/api/http.html) and [`node-fetch`](https://github.com/node-fetch/node-fetch), return a 401 response (if I don't include an API token in the query parameters) or a 200 response (if I do include said API token); I see this behavior when using `curl` as well.

#### Actual behavior

(All commands are executed in a node shell using `node --experimental-repl-await`).

```
const got = require('got');
const url = 'https://api.pipedrive.com/v1/persons/search';
await got(url);
```

throws the following error

```
Uncaught HTTPError: Response code 403 (Forbidden)
    at Request.<anonymous> (got/dist/source/as-promise/index.js:117:42)
    at processTicksAndRejections (internal/process/task_queues.js:93:5) {
  code: undefined,
  timings: {
    start: 1615320853825,
    socket: 1615320853828,
    lookup: 1615320853926,
    connect: 1615320853943,
    secureConnect: 1615320853979,
    upload: 1615320853979,
    response: 1615320854006,
    end: 1615320854012,
    error: undefined,
    abort: undefined,
    phases: {
      wait: 3,
      dns: 98,
      tcp: 17,
      tls: 36,
      request: 0,
      firstByte: 27,
      download: 6,
      total: 187
    }
  }
}
```

. Compare this to `http` and `node-fetch`:

```
const http = require('http');
const getStream = require('get-stream);
https.get(url, async response => { const body = await getStream(response); console.log(body) })
// {"success":false,"error":"unauthorized access","errorCode":401}

const fetch = require('node-fetch'); // v2.6.1
const res = await fetch(url);
await res.json();
// { success: false, error: 'unauthorized access', errorCode: 401 }
```

Finally, using `curl`:

```
> curl 'https://api.pipedrive.com/v1/persons/search'
{"success":false,"error":"unauthorized access","errorCode":401}
```

If I include an `api_token` query parameter in the request url (per the [docs](https://pipedrive.readme.io/docs/core-api-concepts-authentication)), the requests made using `node-fetch`, `http`, and `curl` return a 200 response; `got` continues to return a 403 response in this case. 

#### Expected behavior

That requests made using `got` also return a 401 or a 200 response, depending on the url. I'm wondering if I'm missing something obvious w.r.t. the headers I need to send alongside the request; at the same time, the other approaches I've tested with seem to work out of the box.

#### Code to reproduce

See above.

#### Checklist

- [x] I have read the documentation.
- [x] I have tried my code with the latest version of Node.js and Got.

**Comment by sindresorhus** [comment, marked as answer]
Try setting a `user-agent` header.

**Reply by sbalaji1996** [reply]
Ah, yep, that does the trick. Thanks!
